test/Core.Test/Repositories/EntityFramework/EqualityComparers/InstallationCompare.cs

```diff
@@ -1,14 +1,14 @@
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using Bit.Core.Models.Table;
 
 namespace Bit.Core.Test.Repositories.EntityFramework.EqualityComparers
 {
-    public class InstallationCompare: IEqualityComparer<Installation>
+    public class InstallationCompare : IEqualityComparer<Installation>
     {
         public bool Equals(Installation x, Installation y)
         {
-            return  x.Email == y.Email &&
+            return x.Email == y.Email &&
             x.Key == y.Key &&
```

Comment:
I can't say I'm fond of not indenting `return` statements when they go to the next line, but at least it's in a block I guess.

Reply:
Yea ... it seems dotnet format doesn't handle formatting for these scenarios. I'm not going to manually change it though and will be something we can touch upon next time we edit that file.